content/docs/next/upgrades/upgrading-etcd.md

```diff
@@ -21,8 +21,8 @@ This section contains documents specific to upgrading etcd clusters and applicat
 
 
 [migrate-apps]: ../op-guide/v2-migration
-[upgrade-cluster]: upgrade_3_0
 [upgrade-3-1]: upgrade_3_1
 [upgrade-3-2]: upgrade_3_2
 [upgrade-3-3]: upgrade_3_3
 [upgrade-3-4]: upgrade_3_4
+[upgrade-cluster]: upgrade_3_0
```

Comment:
Hmm, I'd inline these definitions (leaving `migrate-apps`). They don't buy us anything AFAICT. E.g.:

    * [Upgrade etcd from 3.0 to 3.1][upgrade-3-1]

becomes

    * [Upgrade etcd from 3.0 to 3.1](upgrade_3_1)

which is just as readable, and we avoid introducing an extra link definition.

Reply:
Agreed, and updated.